File: RemoteServer/src/RemoteServer/Library/LibraryItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RemoteServer.Library
{
    public class LibraryItem
    {
        public String ItemId { get; set; }
        public String Artist { get; set; }
        public String Album { get; set; }
        public String Title { get; set; }
        public int Duration { get; set; }
        public int Played { get; set; }
        public int Voted { get; set; }
        public int TrackNumber { get; set; }
        public int Rating { get; set; }
        public double Toplist { get; set; }
        public String CoverUrl { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/underscoreresearch/mauritzremote/library/LibraryItem.java
package com.underscoreresearch.mauritzremote.library;

import lombok.Data;

/**
 * Created by henri on 1/29/2017.
 */

@Data
public class LibraryItem
{
    public String itemId;
    public String artist;
    public String album;
    public String title;
    public int duration;
    public int played;
    public int voted;
    public int trackNumber;
    public int rating;
    public double toplist;
    public String coverUrl;
}
